src/main/scala/com/gu/salesfoce/messageHandler/ApiResponse.scala

```diff
@@ -0,0 +1,46 @@
+package com.gu.salesfoce.messageHandler
+
+import java.io.{ OutputStream, OutputStreamWriter }
+import com.gu.salesfoce.messageHandler.ResponseWriters._
+
+import com.gu.salesfoce.messageHandler.ResponseModels.{ ApiResponse, Headers }
+import play.api.libs.json.{ Json, Writes }
+
+object ResponseModels {
+
+  case class Headers(contentType: String = "text/xml")
+
+  case class ApiResponse(statusCode: String, headers: Headers, body: String)
+
+}
+
+object ResponseWriters {
+
+  implicit val headersWrites = new Writes[Headers] {
+    def writes(headers: Headers) = Json.obj(
+      "Content-Type" -> headers.contentType)
+  }
+
+  implicit val responseWrites = new Writes[ApiResponse] {
+    def writes(response: ApiResponse) = Json.obj(
+      "statusCode" -> response.statusCode,
+      "headers" -> response.headers,
+      "body" -> response.body)
+  }
+
+}
+
+object APIGatewayResponse extends Logging {
+
+  def outputForAPIGateway(outputStream: OutputStream, response: ApiResponse): Unit = {
+    val writer = new OutputStreamWriter(outputStream, "UTF-8")
+    val jsonResponse = Json.toJson(response)
+    logger.info(s"Response will be: \n ${jsonResponse.toString}")
```

Comment:
I think in this case it would be more useful to log the response code, and the contactIds from the message we are responding to (rather than the full response). In the majority of cases, presumably the response will just be the successful ack / XML blob which doesn't help us much - we really want to know exactly what contact we told Salesforce that we managed to queue an update for...